### PR TITLE
Test with 2.085 and pin DMD to 2.086.0

### PR DESCRIPTION
**Pull request by Geod24**
https://issues.dlang.org/show_bug.cgi?id=20022

### CODE REVIEW

File: source/scpd/types/XDRBase.d [hunk 1]
```diff
@@ -30,7 +30,7 @@ import scpd.Util;
 import geod24.bitblob;
 
 
-extern(C++, xdr):
+extern(C++, `xdr`):
```
Comment (AndrejMitrovic) [inline source/scpd/types/XDRBase.d:33]: the term you're looking for is FUBARed

**Review by AndrejMitrovic — APPROVED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -1,8 +1,10 @@
 language: d
 
 d:
-  - dmd
+  - dmd-2.086.1 # Pinned due to https://issues.dlang.org/show_bug.cgi?id=20022
   - ldc
+  - dmd-2.085.1
+  - ldc-1.14.0
```
Comment (AndrejMitrovic) [inline .travis.yml]: it looks like this one fails consistently on a Mac.

**Comment by Geod24** [comment]
Updated to LDC-1.15 >.>

**Comment by Geod24** [comment]
Yeah I gave up on older LDCs

**Pull request merged**